modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -834,19 +846,22 @@ protected function register_controls() {
 			'tab' => Controls_Manager::TAB_STYLE,
 		] );
 
+		$styling_block_start = '--n-tabs-title-direction: column; --n-tabs-icon-order: initial; --n-tabs-title-justify-content-toggle: center; --n-tabs-title-align-items-toggle: initial;';
+		$styling_block_end = '--n-tabs-title-direction: column; --n-tabs-icon-order: 1; --n-tabs-title-justify-content-toggle: center; --n-tabs-title-align-items-toggle: initial;';
+
 		$this->add_responsive_control( 'icon_position', [
 			'label' => esc_html__( 'Position', 'elementor' ),
 			'type' => Controls_Manager::CHOOSE,
 			'options' => [
-				'top' => [
+				'block-start' => [
```

Comment:
@rami-elementor Can you please check this?

Reply:
@hein-obox same here, use not only `block-` prefix but also `inline-` prefix.

Reply:
Updated

Reply:
@rami-elementor Implemented

Reply:
@hein-obox Please see the "tabs_direction" control and update its values too.